eip/EIPS/eip-4337.md

```diff
@@ -391,78 +451,104 @@ The result `SHOULD` be set to true if and only if the request passed simulation
     entryPoint // address
   ]
 }
-
-# Response
+```
+Response:
+```
 {
   "jsonrpc": "2.0",
   "id": 1,
   "result": true
 }
 ```
-
-`eth_supportedEntryPoints`
-
-eth_supportedEntryPoints returns an array of the entryPoint addresses supported by the client. The first element of the array `SHOULD` be the entryPoint addressed preferred by the client.
-
-```json=
-# Request
+#### Example failure responses:
+```json
 {
   "jsonrpc": "2.0",
   "id": 1,
-  "method": "eth_supportedEntryPoints",
-  "params": []
+  "error": {
+    "message": "Wallet didn't pay prefund",
+    "code": -33000
+  }
 }
+```
 
-# Response
+```json
 {
   "jsonrpc": "2.0",
   "id": 1,
-  "result": [
-    "0xcd01C8aa8995A59eB7B2627E69b40e0524B5ecf8",
-    "0x7A0A0d159218E6a2f407B99173A2b12A6DDfC2a6"
-  ]
+  "error": {
+    "message": "not staked",
+    "paymaster": "0x123456789012345678901234567890123456790",
+    "code": -33000
```

Comment:
-33000 is not in the list and is also duplicated in an above example (not paying prefund).
I think the list above doesn't have a code for "paymaster not sufficiently staked", so this could be it.